### PR TITLE
chore(): changelog for covalent@rc.0

### PR DESCRIPTION
**Pull request by emoralesb05**
## Description

Changelog and notification updates for rc release.

#### General Tests for Every PR

- [ ] `ng serve --aot` still works.
- [ ] `npm run lint` passes.
- [ ] `npm test` passes and code coverage is not lower.
- [ ] `npm run build` still works.

### CODE REVIEW

File: src/app/components/toolbar/toolbar.component.html [hunk 1]
```diff
@@ -29,9 +29,16 @@ <h4 matLine><span class="text-wrap">{{item.description}}</span></h4>
             </a>
             <mat-divider></mat-divider>
           </ng-template>
-          <a mat-list-item href="https://github.com/Teradata/covalent/blob/develop/package.json#L63-L74" target="_blank">
+          <a mat-list-item href="https://github.com/Teradata/covalent/blob/develop/docs/CHANGELOG.md" target="_blank">
+            <mat-icon matListAvatar>receipt</mat-icon>
+            <h4 matLine>
+              <span class="text-wrap">Bigfixed, improvements and features</span>
```
Comment (aciccarello) [inline src/app/components/toolbar/toolbar.component.html]: Is this supposed to say "bugfixes"?

File: src/app/components/toolbar/toolbar.component.ts [hunk 2]
```diff
@@ -17,10 +17,20 @@ import { getDirection, setDirection } from '../../utilities/direction';
...
+      icon: 'gradient',
+      route: '/docs/utility-sass-mixins',
+      title: 'Utility Style Mixins',
+    }, {
+      description: 'Override the typography',
+      icon: 'font_download',
+      route: '/docs',
+      title: 'Custom Typography',
+    }, {
+      description: 'New ones and now using StackBlitz',
```
Comment (aciccarello) [inline src/app/components/toolbar/toolbar.component.ts]: Without seeing the title "New ones" is confusing. Perhaps "New templates" would be more clear even if it is redundant.

File: docs/CHANGELOG.md [hunk 3]
```diff
@@ -1,3 +1,44 @@
...
+- `@angular@5.0.0` support
+- `@angular/material@5.0.0-rc.0` support
+- Real stand alone modules with no dependency to external css styles
+- Create style mixins to pick and choose which utility classes you want
+- Configurable typography for all Covalent components
+- More templates and migration to StackBlitz
+- Bugfixes and improvements
+
+## Bug Fixes
+* **docs:** add missing (rowClick) event to data-table documentation [b2d8cc229895aef88fe35df1580fc3c9e7a5beef](https://github.com/Teradata/covalent/commit/b2d8cc229895aef88fe35df1580fc3c9e7a5beef))
```
Comment (aciccarello) [inline docs/CHANGELOG.md]: I'm seeing an extra `)` in the rich diff

File: src/app/components/toolbar/toolbar.component.html (hunk 1 above)
Comment (emoralesb05) [reply src/app/components/toolbar/toolbar.component.html]: whoops hahha my bad

**Pull request merged**